### PR TITLE
User views

### PR DESCRIPTION
**Pull request by blakekrammes**
Depends on the session branch #80

### CODE REVIEW

File: apps/field/src/users/UserForm.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    displayFirstName,
+    displayLastName,
+    email,
+    phone,
+    timezone,
+    bio,
+  } = record;
+  return (
+    <SimpleForm
+      {...omit(props, ['hasEdit', 'hasList', 'hasShow', 'hasCreate'])}
```
Comment (CarsonF) [inline apps/field/src/users/UserForm.tsx]: Why are we omitting these?

File: apps/field/src/users/UserForm.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      record={record}
+    >
+      {create || realFirstName?.canRead ? (
+        <TextInput
+          source={create ? 'realFirstName' : 'realFirstName.value'}
+          label="First Name"
+          disabled={create ? false : !(realFirstName?.canEdit ?? false)}
+          validate={[validateLength, requireField]}
+        />
+      ) : null}
```
Comment (CarsonF) [inline apps/field/src/users/UserForm.tsx]: We should see about abstracting this in future.

File: apps/field/src/users/UserForm.tsx (hunk 1 above)
Comment (blakekrammes) [reply apps/field/src/users/UserForm.tsx]: An error is logged to the console. I'm guessing they're props that Edit or create would use, but not SimpleForm

File: apps/field/src/users/UserForm.tsx (hunk 2 above)
Comment (blakekrammes) [reply apps/field/src/users/UserForm.tsx]: Yeah. I was thinking that too.

File: apps/field/src/users/UserForm.tsx (hunk 1 above)
Comment (CarsonF) [reply apps/field/src/users/UserForm.tsx]: Ah I see. I still want to see if I can pull these into a context sometimes so less has to be passed around. https://github.com/marmelab/react-admin/issues/4243

Comment (blakekrammes) [reply apps/field/src/users/UserForm.tsx]: Got it. Cool.

**Pull request merged**